recipe/meta.yaml

```diff
@@ -26,10 +26,10 @@ requirements:
     - lxml
     - networkx
     - tinydb
-    - openff-toolkit >=0.9.0,<0.11.0
-    - openff-forcefields >=1.2.0
     - validators
 
+  run_constrained:
+    - openff-toolkit >=0.9.0
```

Comment:
IIUC one could use the other parts of `openmmforcefields` and ignore SMIRNOFF world altogether. This would enable that while also ensuring a goofy old version isn't pulled down

Reply:
This could also just pull down `openff-toolkit-base`, since everything besides RDKit is either pulled in here or not needed for this functionality. But I don't think the benefit is worth the (small) risk of chaos

https://github.com/conda-forge/openff-toolkit-feedstock/blob/52c62b66e393039cf3b256c6b0b9b09b503effe0/recipe/meta.yaml#L57-L65

Reply:
Do we need `openff-toolkit` still in requires or is it enough that it is in run constrained?

Reply:
That's a matter of if you want it pulled down in the vanilla install - `run_constrained` adds constraints without adding it as a dependency, i.e. `conda install openmmforcefields` will not install the toolkit. If you'd rather the toolkit just be pulled down whenever `openmforcefields` is installed I can just move it back to where it was

Reply:
Let's just move it back, I worry since it has been in the vanilla install as a requirement that removing it could cause issues